apps/server/src/modules/user-login-migration/service/user-login-migration.service.spec.ts

```diff
@@ -523,100 +523,96 @@ describe(UserLoginMigrationService.name, () => {
 	describe('setMigrationMandatory', () => {
 		describe('when migration is set to mandatory', () => {
 			const setup = () => {
-				const schoolId: EntityId = new ObjectId().toHexString();
-
-				const targetSystemId: EntityId = new ObjectId().toHexString();
-
-				const userLoginMigrationDO: UserLoginMigrationDO = userLoginMigrationDOFactory.buildWithId({
-					targetSystemId,
-					schoolId,
+				const userLoginMigration: UserLoginMigrationDO = userLoginMigrationDOFactory.buildWithId({
 					startedAt: mockedDate,
-					mandatorySince: undefined,
+					mandatorySince: mockedDate,
```

Comment:
this should be undefined again, since the test wants to test, if it has changed from undefined to mandatory